doc/library/gpuarray/dnn.txt

```diff
@@ -1,7 +1,7 @@
 .. _libdoc_gpuarray_dnn:
 
 ===========================================
-:mod:`theano.sandbox.gpuarray.dnn` -- cuDNN
+:mod:`gpuarray.dnn` -- cuDNN
```

Comment:
All other modules omit the theano prefix in the doc, I wrote it this way to stay consistent.